bin/ch/WScriptJsrt.cpp

```diff
@@ -1551,19 +1607,23 @@ HRESULT WScriptJsrt::ModuleMessage::Call(LPCSTR fileName)
 
             if (FAILED(hr))
             {
-                if (!HostConfigFlags::flags.MuteHostErrorMsgIsEnabled)
+                // check if have it registered
+                AutoString *data;
```

Comment:
Since `data` is a pointer, I don't think the AutoString destructor will run -- is that an issue?

Reply:
It shouldn't run by design and intention. See constructor. It takes over the ownership (there is a comment). Source code map lifetime == process lifetime.